namui-prebuilt/src/dubu_gat.rs

```diff
@@ -0,0 +1,195 @@
+// NOTE: This is unused file for now. I will update it as main when GAT is ready on rust.
```

Comment:
Please do not review this code yet. I will use this code later when GAT is ready for rust.